test/CloudNative.CloudEvents.UnitTests/CloudEventsSpecVersionTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2021 Cloud Native Foundation. 
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace CloudNative.CloudEvents.UnitTests
+{
+    public class CloudEventsSpecVersionTest
+    {
+        [Theory]
+        [InlineData(null)]
+        [InlineData("bogus")]
```

Comment:
May want to throw in "1" for good measure.

Reply:
Done.